NMS-14633: Add alt text to Operation (pt. 2) (#5677)
OpenNMS/opennms: docs/modules/operation/pages/deep-dive/performance-data-collection/snmp-property-extenders/property-extenders.adoc

----

Most SNMP property extenders make it possible to include string properties from a "source" MIB table that is indexed differently from the table containing most of the relevant data.

> For the purpose of configuring property extenders, the table containing the majority of the data (and into which we want to include the string properties) is called the "target table", and the table containing the string property is called the "source table".

Reviewer: ```suggestion
When configuring property extenders, the table containing the majority of the data (and into which we want to include the string properties) is called the "target table", and the table containing the string property is called the "source table".
```
